index.html

```diff
@@ -30,6 +30,12 @@ <h5>Select category</h5>
           <button value="dg_large_kw" type="button" class="btn btn-light">Large DG</button>
           <button value="dg_small_kw" type="button" class="btn btn-light">Small DG</button>
         </div>
+        <br /><br />
+        <h5>Select status</h5>
+        <div id="status-select" class="btn-group flex-wrap" role="group" aria-label="Status">
+          <button value="energized" type="button" class="btn btn-light active">Energized</button>
```

Comment:
This is an issue with your other buttons as well, but worth noting a toggle button like this should use [`aria-pressed`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed) to indicate to screen readers when it's active or not. Accessibility might just be a separate task though

Reply:
i will open this up as a separate issue to address